cli/build.js

```diff
@@ -2,7 +2,7 @@
 'use strict';
 
 const fs = require('fs-extra');
-const merge = require('lodash.merge');
+const merge = require('../utils/merge-utils');
```

Comment:
If we go this route, we can probably forego the added `-utils` suffix on the file, since it's already in the utils folder.